### PR TITLE
Spring 2021 Newsletter

### PR DESCRIPTION
**Pull request by 12xx12**
I wrote a spring 2021 newsletter.

I'm not quite sure what to do about the discussion-id in beginning

### CODE REVIEW

**Comment by tigerw** [comment]
I have two PRs, we can hopefully get those in before this goes live

**Comment by 12xx12** [comment]
Knock yourself out - I just wanted to get this started. Maybe on a more regular basis

File: news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: 'Cuberite Newsletter Spring 2021'
+discussion-id: 3306
```
Comment (mathiascode) [inline news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md]: You should now be able to create a draft in https://forum.cuberite.org/forum-7.html. The thread ID can then be included here.

Comment (12xx12) [reply news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md]: Ok - and when this PR is merged I release the forum post 💡

File: news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ - Thanks to all the new contributors this period, as we now have 3 new 
+   entries in the CONTRIBUTORS file:
+   - 12xx12
+   - dImrich
+   - KingCol13
+
+Contribute
+----------
+
+ - If you are interested in developing Cuberite's core or plugins,
```
Comment (mathiascode) [inline news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md]: These links could be clickable, see: https://github.com/cuberite/cuberite.github.io/commit/82b6d97804b531a19057f7e925703e3a100f297a

**Comment by 12xx12** [comment]
I think we could maybe merge the small generator PRs also cuberite/cuberite#5129

File: news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+project as well as what's gone on since the last update!
+
+General News
+------------
+
+  - Progress for protocol versions greater than 1.12 is still encouraging.
+    Connections from 1.13 clients are accepted but not realy playable. We're working
+    on the needed changes for the new block state format.
+    [#5151](https://github.com/cuberite/cuberite/pull/5151)
+  - Due to a few changes in the generator it's recommended to regenrate the world
```
Comment (bearbin) [inline news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md]:
```suggestion
  - Due to a few changes in the generator it's recommended to regenerate the world
```

File: news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+   - [#5134](https://github.com/cuberite/cuberite/pull/5134):
+     Adds mushrooms to the generator and fixed up the roofed forest
+   - [#5179](https://github.com/cuberite/cuberite/pull/5179):
+     Teleport off of mount when throwing enderpearl
+
+ - Thanks to all the new contributors this period, as we now have 3 new 
+   entries in the CONTRIBUTORS file: 
+   - 12xx12 
+   - dImrich 
+   - KingCol13
```
Comment (bearbin) [inline news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md]: This needs updating.

**Review by bearbin — COMMENTED** [review]
Looks good aside from two small points, I will send out the newsletter by email once this is merged in.

File: news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md (hunk 4 above)
Comment (12xx12) [reply news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md]: Does it - I thought I got everyone. Gonna check again

Comment (bearbin) [reply news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md]: ProjectBM
pwnOrbitals
theophriene
tympaniplayer (Nate Palmer)

Comment (12xx12) [reply news/_posts/2021-04-08-cuberite-spring-2021-newsletter.md]: mh... yeah.. my bad. I thought I got everyone. I used the web blame feature. Well Thanks :D

**Comment by 12xx12** [comment]
@tigerw Do you still want to wait?

**Comment by 12xx12** [comment]
Or does anyone have another PR worht mentioning?

**Comment by 12xx12** [comment]
Oh - and we need to update the date on the file when we finally release the newsletter

**Comment by tigerw** [comment]
For me, I don't mind either way

**Comment by 12xx12** [comment]
I updated the date of the file and the contrubutors and fixed two littly typos. So i think we are ready to merge

**Comment by bearbin** [comment]
LGTM, I will send out the newsletter. @12xx12 you created the forum draft right?

**Comment by 12xx12** [comment]
Yes i did. Is it time to publish?

**Comment by 12xx12** [comment]
wait a second. Got to find it again

**Comment by bearbin** [comment]
Yep, since this is merged it's on the website already so the forum thread can go public.

**Comment by 12xx12** [comment]
done 👍

**Comment by bearbin** [comment]
And the corresponding tweet and email have been sent out.

**Pull request merged**